File: Python/01.ProgrammingBasics/06.Conditional-Statements-Adv-Exercise/05.Journey/_05.Journey.py
budget = float(input())
season = input()

destination = ""
vacation_type = ""
spent_money = 0

if budget <= 100:
    destination = "Bulgaria"
    if season == "summer":
        vacation_type = "Camp"
        spent_money = budget * 0.3
    elif season == "winter":
        vacation_type = "Hotel"
        spent_money = budget * 0.7
elif 100 < budget <= 1000:
    destination = "Balkans"
    if season == "summer":
        vacation_type = "Camp"
        spent_money = budget * 0.4
    elif season == "winter":
        vacation_type = "Hotel"
        spent_money = budget * 0.8
elif budget > 1000:
    destination = "Europe"
    vacation_type = "Hotel"
    spent_money = budget * 0.9

print(f"Somewhere in {destination}")
print(f"{vacation_type} - {spent_money:.2f}")

File: C++/01-Basics/03.3-ConditionalStatementsAdvЕxercise/05.Journey/05.Journey.cpp
#include <iostream>
#include <string>
#include <iomanip>

using namespace std;

int main() {
    double budget;
    string season;
    cin >> budget >> season;

    string destination;
    string vacationType;
    double expenses;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season == "summer") {
            vacationType = "Camp";
            expenses = 0.30 * budget;
        }
        else if (season == "winter") {
            vacationType = "Hotel";
            expenses = 0.70 * budget;
        }
    }
    else if (budget <= 1000) {
        destination = "Balkans";
        if (season == "summer") {
            vacationType = "Camp";
            expenses = 0.40 * budget;
        }
        else if (season == "winter") {
            vacationType = "Hotel";
            expenses = 0.80 * budget;
        }
    }
    else {
        destination = "Europe";
        vacationType = "Hotel";
        expenses = 0.90 * budget;
    }

    cout << "Somewhere in " << destination << endl;
    cout << vacationType << " - " << fixed << setprecision(2) << expenses << endl;

    return 0;
}
